File: src/main/java/com/janson/guess/Game.java
package com.janson.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret) {
            System.out.print("請輸入一個數字：");
            number = scanner.nextInt();
            if (number < secret) System.out.println("您猜的是" + number + ",請再猜大一點"); else System.out.println("您猜的是" + number + ",請再猜小一點");
            System.out.println("");
        }
        System.out.println("恭喜答對了！");
//        int i = 0;
//        do {
//            System.out.println(i);
//            i+=1;
//        }while (i<3);
//        System.out.println(i);
    }
}

File: src/main/kotlin/com/janson/guess/Game.kt
package com.janson.guess

import java.util.*

fun main() {
    val secret = Random().nextInt(10) + 1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret) {
        print("請猜一個數字：")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        if (number < secret) println("您猜的是$number,請再猜大一點") else println("您猜的是$number,請再猜小一點")
    }
    println("恭喜猜對了，謎底是 $secret")
}